app/javascript/chat/components/createChatModal.jsx

```diff
@@ -0,0 +1,65 @@
+import { h } from 'preact';
+import { useState } from 'preact/hooks';
+import PropTypes from 'prop-types';
+import { Modal } from '../../crayons/Modal';
```

Comment:
refactor (non-blocking): You can use the webpack alias for crayons instead.

```suggestion
import { Modal } from '@crayons/Modal';
```